charts/prl-ccd-definitions/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Private Law - CCD Definitions
 name: prl-ccd-definitions
-version: 0.0.18
+version: 0.0.19
```

Comment:
We do not need this bump

Reply:
@AlokDatta Reverted to 0.0.18, can you please check again.